.buildkite/react-native-android-pipeline.yml

```diff
@@ -152,204 +152,222 @@ steps:
         depends_on: "rn-0-60-apk"
         timeout_in_minutes: 60
         plugins:
-          artifacts#v1.5.0:
+          artifacts#v1.9.0:
             download: "build/rn0.60.apk"
             upload: ./test/react-native/maze_output/**/*
-          docker-compose#v3.9.0:
+          docker-compose#v4.7.0:
             pull: react-native-maze-runner
             run: react-native-maze-runner
-            use-aliases: true
+            service-ports: true
             command:
             - --app=build/rn0.60.apk
-            - --farm=bs
-            - --device=ANDROID_11_0
+            - --farm=bb
+            - --device=ANDROID_11
```

Comment:
I think we can chain together a number of versions here to make better use of resources.  Maybe ANDROID_10|....|ANDROID_13?